compiler/cpp/src/generate/t_rs_generator.cc

```diff
@@ -262,27 +218,12 @@ string t_rs_generator::rs_autogen_comment() {
 }
 
 string t_rs_generator::rs_imports() {
-  return string(
-    "#[allow(unused_imports)]\n"
-    "use std::collections::{HashMap, HashSet};\n"
-    "use std::rc::Rc;\n"
-    "use std::cell::RefCell;\n"
-    "use thrift::processor::Processor;\n"
-    "use thrift::protocol::{Protocol, MessageType, Type};\n"
-    "use thrift::transport::Transport;\n"
-    "use thrift::protocol::{Readable, Writeable, ProtocolHelpers};\n"
-    "use thrift::TResult;\n"
-    "#[allow(unused_imports)]\n"
-    "use thrift::ThriftErr;\n"
-    "#[allow(unused_imports)]\n"
-    "use thrift::ThriftErr::*;\n"
-    "#[allow(unused_imports)]\n"
-    "use thrift::protocol::Error;\n"
-    "#[allow(unused_imports)]\n"
-    "use thrift::protocol::FromNum;\n"
-  );
+  return string("#![allow(unused_mut, dead_code, non_snake_case)]\n") +
```

Comment:
My C++ isn't great but I don't think you need the `string()` here. If not, can you remove the one in `rs_autogen_comment` too?